resources/lang/de_DE/default.po

```diff
@@ -2397,6 +2395,13 @@ msgstr "DECT"
 msgid "angeltypes.email"
 msgstr "E-Mail"
 
+msgid "angeltypes.hide.on.shift.view"
+msgstr "Auf Schicht Ansicht ausblenden"
```

Comment:
Eventually
```suggestion
msgstr "Auf Schicht-Ansicht ausblenden"
```
(and other occurrences too)

Reply:
done